protobuf/types/execution.proto

```diff
@@ -119,4 +120,22 @@ message Execution {
   int64 blockHeight = 15 [
     (gogoproto.moretags) = 'hash:"-"'
   ];
+
+  // Emitter is a runner that proposed an execution.
+  message Emitter {
+    // Emitter's address.
+    string address = 1 [
+      (gogoproto.moretags) = 'hash:"-" validate:"accaddress"'
+    ];
+
+    // Block height when this emitter proposed the execution.
+    int64 blockHeight = 2 [
```

Comment:
not really sure block height is useful high now, I would keep it simple, it would remove some complexity in the code